configure.py

```diff
@@ -609,6 +609,13 @@
     default=False,
     help='compile V8 with minimal optimizations and with runtime checks')
 
+parser.add_option('--load-builtin-modules-from-disk',
```

Comment:
Just realized this should imply `--without-node-snapshot` and `--without-node-code-cache`.